test-utils/test/shared/act.test.js

```diff
@@ -516,4 +516,56 @@ describe('act', () => {
 			});
 		});
 	});
+
+	describe('act function with finish implementations', function () {
```

Comment:
Nit: should generally be using arrow functions, as the other examples in this file show.